sdk/core/core/inc/az_span.h

```diff
@@ -26,7 +26,7 @@
 #include <_az_cfg_prefix.h>
 
 /**
- * A span is a "view" over a byte buffer that represents a contiguous region of memory. It contains
+ * @brief A span is a "view" over a byte buffer that represents a contiguous region of memory. It contains
```

Comment:
Nit: In some places the pattern is "A <type> is a ..." and in other cases the brief is "Represents ...."

Reply:
I don't know if consistency here is that important given different people end up writing it differently. But you are right, and I will fix this particular one since I am already touching it in this PR.